src/pages/index.tsx

```diff
@@ -1,18 +1,125 @@
 import { useManager } from "@cosmos-kit/react";
-import { useEffect } from "react";
+import { RouteResponse, SkipRouter, TxStatusResponse } from "@skip-router/core";
+import { differenceInMinutes, parseISO } from "date-fns";
+import { useState } from "react";
 
 import { SwapWidget } from "@/components/SwapWidget";
 import { WalletModalProvider } from "@/components/WalletModal";
 import {
   failTxHistory,
   successTxHistory,
+  TxHistoryItem,
+  updateTxStatus,
   useTxHistory,
 } from "@/context/tx-history";
 import { useSkipClient } from "@/solve";
 import { useInterval } from "@/utils/hooks";
 import { queryClient } from "@/utils/query";
 import { getBalancesByChain } from "@/utils/utils";
 
+function routeHasAxelarTransfer(route: RouteResponse): boolean {
```

Comment:
I imagine a lot of these functions might be useful elsewhere in our code at some point, but until then I'll just leave them in the same file they're used